### PR TITLE
feat(contract_manager): add fuel mainnet config

### PR DESCRIPTION
**Pull request by cctdaniel**

### CODE REVIEW

File: contract_manager/store/contracts/FuelPriceFeedContracts.yaml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 - chain: fuel_testnet
-  address: "0xe31e04946c67fb41923f93d50ee7fc1c6c99d6e07c02860c6bea5f4a13919277"
+  address: "0x25146735b29d4216639f7f8b1d7b921ff87a1d3051de62d6cceaacabeb33b8e7"
```
Comment (cctdaniel) [inline contract_manager/store/contracts/FuelPriceFeedContracts.yaml:2]: drive-by: update to the latest testnet deployment

**Review by ali-bahjati — APPROVED** [review]
